react-challenge.html

```diff
@@ -40,7 +42,8 @@
 
         render(){
           return(
-
+            //Placed carrot brackets as shown in the error message from the console in the browser but more errors come up.
+            <>
```

Comment:
Hey React Fragments! Good work here and good pseudo code. and Yes more errors do show up and working through errors one at a time is definitely part of Dev life.